### PR TITLE
Update to sdk@13 partially

### PR DESCRIPTION
**Pull request by davidyuk**
This PR is supported by the Æternity Crypto Foundation

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -3,24 +3,21 @@ on: [pull_request]
 jobs:
   main:
     runs-on: ubuntu-latest
-    env:
-      CYPRESS_PLUGIN_VISUAL_REGRESSION_UPDATE_IMAGES: ${{contains(github.event.head_commit.message, ' [update screenshots]')}}
```
Comment (davidyuk) [inline .github/workflows/test.yml:7]: I find out that there is no one-liner to get the last commit message, so I'm switching to a different strategy: commit e2e artifacts, and then the developer needs to manually pick the right screenshots and restart tests.
https://github.com/orgs/community/discussions/28474

File: cypress.config.js [hunk 2]
```diff
@@ -19,6 +21,28 @@ module.exports = defineConfig({
...
+        fs.linkSync(defaultVideoPath, videoPath);
+
+        if (browser.family === 'chromium' && browser.name !== 'electron') {
+          launchOptions.args.push(`--use-file-for-fake-video-capture=${videoPath}`);
+        }
+        return launchOptions;
+      });
+
+      on('task', {
+        changeVideoSource(videoSource) {
```
Comment (davidyuk) [inline cypress.config.js:39]: based on https://github.com/cypress-io/cypress/issues/21188#issuecomment-1289579740

File: src/components/mobile/ConfirmTransactionSignModal.vue [hunk 3]
```diff
@@ -60,7 +60,7 @@
 
 <script>
 import { mapState } from 'vuex';
-import { SCHEMA } from '@aeternity/aepp-sdk';
+import { Tag } from '@aeternity/aepp-sdk-next';
```
Comment (thepiwo) [inline src/components/mobile/ConfirmTransactionSignModal.vue:63]: why next?

Comment (davidyuk) [reply src/components/mobile/ConfirmTransactionSignModal.vue:63]: I would rename it back to `@aeternity/aepp-sdk` after finishing the update

**Comment by dincho** [comment]
What's the status of this PR?

**Pull request merged**